copernicus_services/hrsi-rlie-s1-quality/script.js

```diff
@@ -0,0 +1,23 @@
+//VERSION=3
+function setup() {
+  return {
+      input: ["QC", "dataMask"],
+      output: {bands: 4,
+      sampleType: "Auto"
+      }
+  };
+
+}
+
+const map = [
+  [0, 0x5DA400],  //0 - High quality
+  [1, 0xBDBD5B],  //1 - Medium quality
+  [2, 0xFFC000],  //0 - Low quality
+  [3, 0xFF0000],  //1 - Minimal quality
+  [255, 0xFFFFFF],  //255 - No data
+];
+
+const visualizer = new ColorMapVisualizer(map);
+
+function evaluatePixel(sample) {
+return [visualizer.process(sample.RLIE)[0], visualizer.process(sample.RLIE)[1], visualizer.process(sample.RLIE)[2], sample.dataMask]};
```

Comment:
To make this better have a look at this [script](https://github.com/sentinel-hub/custom-scripts/blob/master/copernicus_services/hrsi-psa-psa-quality/script.js). Doing it like this avoids calling `.process` unnecessary amount of times. This should be changed in all scripts:

```suggestion
  let rgbVis = visualizer.process(sample.QC);
  return rgbVis.concat(sample.dataMask);
```

With all of the QC scripts you also forgot to change the band name from `sample.RLIE` to `RLIE.QC`